scripts/lint_python_docstrings.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Lint docstrings only against the the diff to avoid too many errors.
+# Check only production code. Ignore other other errors which are captured by `lint`
+
+# Compare against `main` if no branch was provided
+BRANCH="${1:-main}"
+# Diff of committed changes (shows only changes introduced by your branch
+FILES_WITH_DIFF=`git diff $BRANCH...HEAD --name-only -- rasa`
+NB_FILES_WITH_DIFF=`echo $FILES_WITH_DIFF | grep '\S' | wc -l`
+
+if [ "$NB_FILES_WITH_DIFF" -gt 0 ]
```

Comment:
have to do this because `ruff` errors out if no python files are passed. flake8 didn't complain

(if you have a more elegant approach please suggest it, I don't like what I wrote here 😅 )